src/main/webapp/app/entities/result/result-history.component.ts

```diff
@@ -36,7 +36,8 @@ export class ResultHistoryComponent {
             return 0;
         }
         if (result.resultString && result.resultString.indexOf('failed') !== -1) {
-            return null;
+            // TODO: Check if returning 0 is ok
```

Comment:
@maximilianmeier Can you check if returning 0 is ok?

Reply:
we need the null. i reverted it

Reply:
Did you forget to push that change? 😀

Reply:
yeah. i pushed it